src/main/java/com/mraof/minestuck/util/MSCapabilities.java

```diff
@@ -1,6 +1,8 @@
 package com.mraof.minestuck.util;
 
 import com.mraof.minestuck.Minestuck;
+import com.mraof.minestuck.computer.editmode.EditToolsCapabilityProvider;
+import com.mraof.minestuck.computer.editmode.IEditTools;
 import com.mraof.minestuck.inventory.musicplayer.IMusicPlaying;
 import com.mraof.minestuck.inventory.musicplayer.MusicPlayingCapabilityProvider;
 import com.mraof.minestuck.network.MusicPlayerPacket;
```

Comment:
Wasnt something you changed, but if you feel motivated would you mind removing the unused MusicPlayerPacket import? Feel free to mark this as closed even if you dont